data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_without_consent.py

```diff
@@ -227,12 +248,23 @@ def get_sandbox_tablenames(self):
     if ARGS.list_queries:
         clean_engine.add_console_logging()
         query_list = clean_engine.get_query_list(
-            ARGS.project_id, ARGS.dataset_id, ARGS.sandbox_dataset_id,
-            [(RemoveEhrDataWithoutConsent,)])
+            ARGS.project_id,
+            ARGS.dataset_id,
+            ARGS.sandbox_dataset_id,
+            ARGS.duplicates_table,
+            [(RemoveEhrDataWithoutConsent,)],
+            duplicates_dataset=ARGS.duplicates_dataset,
+            duplicates_table=ARGS.duplicates_table,)
+
         for query in query_list:
             LOGGER.info(query)
     else:
         clean_engine.add_console_logging(ARGS.console_log)
-        clean_engine.clean_dataset(ARGS.project_id, ARGS.dataset_id,
-                                   ARGS.sandbox_dataset_id, ARGS.cutoff_date,
+        clean_engine.clean_dataset(ARGS.project_id,
+                                   ARGS.dataset_id,
+                                   ARGS.sandbox_dataset_id,
+                                   ARGS.cutoff_date,
+                                   duplicates_dataset = ARGS.duplicates_dataset,
+                                   duplicates_table = ARGS.duplicates_table
```

Comment:
This won't work when run as a stand alone module.  `[(RemoveEhrDataWithoutConsent,)]` should come before your optional parameter declarations, as in lines 255-257.

Reply:
I specified kwargs between positionals.